integrations/kubecost/main.py

```diff
@@ -0,0 +1,18 @@
+from typing import Any
+
+from client import KubeCostClient
+from port_ocean.context.ocean import ocean
+
+
+@ocean.on_resync("kubesystem")
+async def on_kubesystem_cost_resync(kind: str) -> list[dict[Any, Any]]:
+    client = KubeCostClient(ocean.integration_config["kubecost_host"])
+    data = await client.get_kubesystem_cost_allocation()
+    return [value for item in data for value in item.values()]
+
+
+@ocean.on_resync("cloud")
+async def on_cloud_cost_resync(kind: str) -> list[dict[Any, Any]]:
+    client = KubeCostClient(ocean.integration_config["kubecost_host"])
+    data = await client.get_cloud_cost_allocation()
+    return [value for item in data for value in item["cloudCosts"].values()]
```

Comment:
why are u suing .values?
how does the object look like?

Reply:
```json
```json
{
    "code": 200,
    "data": {
        "sets": [
            {
                "cloudCosts": {
                    "5hnnev4d0v7mapf09j0v8of0o2": {
                        "properties": {
                            "provider": "AWS",
                            "accountID": "297945954695",
                            "invoiceEntityID": "297945954695",
                            "service": "5hnnev4d0v7mapf09j0v8of0o2"
                        },
                        "window": {
                            "start": "2023-04-30T00:00:00Z",
                            "end": "2023-05-01T00:00:00Z"
                        },
                        "listCost": {
                            "cost": 8.207999999999997,
                            "kubernetesPercent": 0
                        },
                        "netCost": {
                            "cost": 8.207999999999997,
                            "kubernetesPercent": 0
                        },
                        "amortizedNetCost": {
                            "cost": 8.207999999999997,
                            "kubernetesPercent": 0
                        },
                        "invoicedCost": {
                            "cost": 8.207999999999997,
                            "kubernetesPercent": 0
                        }
                    },
                    "AWSBackup": {
                        "properties": {
                            "provider": "AWS",
                            "accountID": "297945954695",
                            "invoiceEntityID": "297945954695",
                            "service": "AWSBackup",
                            "labels": {
                                "name": "khand-dev"
                            }
                        },
                        "window": {
                            "start": "2023-04-30T00:00:00Z",
                            "end": "2023-05-01T00:00:00Z"
                        },
                        "listCost": {
                            "cost": 4e-10,
                            "kubernetesPercent": 0
                        },
                        "netCost": {
                            "cost": 4e-10,
                            "kubernetesPercent": 0
                        },
                        "amortizedNetCost": {
                            "cost": 4e-10,
                            "kubernetesPercent": 0
                        },
                        "invoicedCost": {
                            "cost": 4e-10,
                            "kubernetesPercent": 0
                        }
                    },
                    "AWSCloudTrail": {
                        "properties": {
                            "provider": "AWS",
                            "accountID": "297945954695",
                            "invoiceEntityID": "297945954695",
                            "service": "AWSCloudTrail"
                        },
                        "window": {
                            "start": "2023-04-30T00:00:00Z",
                            "end": "2023-05-01T00:00:00Z"
                        },
                        "listCost": {
                            "cost": 4.9206699999999985,
                            "kubernetesPercent": 0
                        },
                        "netCost": {
                            "cost": 4.9206699999999985,
                            "kubernetesPercent": 0
                        },
                        "amortizedNetCost": {
                            "cost": 4.9206699999999985,
                            "kubernetesPercent": 0
                        },
                        "invoicedCost": {
                            "cost": 4.9206699999999985,
                            "kubernetesPercent": 0
                        }
                    },
                    "AWSCostExplorer": {
                        "properties": {
                            "provider": "AWS",
                            "accountID": "297945954695",
                            "invoiceEntityID": "297945954695",
                            "service": "AWSCostExplorer"
                        },
                        "window": {
                            "start": "2023-04-30T00:00:00Z",
                            "end": "2023-05-01T00:00:00Z"
                        },
                        "listCost": {
                            "cost": 0.26426064520000003,
                            "kubernetesPercent": 0
                        },
                        "netCost": {
                            "cost": 0.26426064520000003,
                            "kubernetesPercent": 0
                        },
                        "amortizedNetCost": {
                            "cost": 0.26426064520000003,
                            "kubernetesPercent": 0
                        },
                        "invoicedCost": {
                            "cost": 0.26426064520000003,
                            "kubernetesPercent": 0
                        }
                    },
                    "AWSELB": {
                        "properties": {
                            "provider": "AWS",
                            "invoiceEntityID": "297945954695",
                            "service": "AWSELB"
                        },
                        "window": {
                            "start": "2023-04-30T00:00:00Z",
                            "end": "2023-05-01T00:00:00Z"
                        },
                        "listCost": {
                            "cost": 43.00682560389998,
                            "kubernetesPercent": 0.8073338296107909
                        },
                        "netCost": {
                            "cost": 43.00682560389998,
                            "kubernetesPercent": 0.8073338296107909
                        },
                        "amortizedNetCost": {
                            "cost": 43.00682560389998,
                            "kubernetesPercent": 0.8073338296107909
                        },
                        "invoicedCost": {
                            "cost": 43.00682560389998,
                            "kubernetesPercent": 0.8073338296107909
                        }
                    },
                    "AWSGlue": {
                        "properties": {
                            "provider": "AWS",
                            "accountID": "297945954695",
                            "invoiceEntityID": "297945954695",
                            "service": "AWSGlue"
                        },
                        "window": {
                            "start": "2023-04-30T00:00:00Z",
                            "end": "2023-05-01T00:00:00Z"
                        },
                        "listCost": {
                            "cost": 0.43269115999999996,
                            "kubernetesPercent": 0
                        },
                        "netCost": {
                            "cost": 0.43269115999999996,
                            "kubernetesPercent": 0
                        },
                        "amortizedNetCost": {
                            "cost": 0.43269115999999996,
                            "kubernetesPercent": 0
                        },
                        "invoicedCost": {
                            "cost": 0.43269115999999996,
                            "kubernetesPercent": 0
                        }
                    },
                    "AWSLambda": {
                        "properties": {
                            "provider": "AWS",
                            "accountID": "297945954695",
                            "invoiceEntityID": "297945954695",
                            "service": "AWSLambda"
                        },
                        "window": {
                            "start": "2023-04-30T00:00:00Z",
                            "end": "2023-05-01T00:00:00Z"
                        },
                        "listCost": {
                            "cost": 0,
                            "kubernetesPercent": 0
                        },
                        "netCost": {
                            "cost": 0,
                            "kubernetesPercent": 0
                        },
                        "amortizedNetCost": {
                            "cost": 0,
                            "kubernetesPercent": 0
                        },
                        "invoicedCost": {
                            "cost": 0,
                            "kubernetesPercent": 0
                        }
                    },
                    "AWSQueueService": {
                        "properties": {
                            "provider": "AWS",
                            "accountID": "297945954695",
                            "invoiceEntityID": "297945954695",
                            "service": "AWSQueueService"
                        },
                        "window": {
                            "start": "2023-04-30T00:00:00Z",
                            "end": "2023-05-01T00:00:00Z"
                        },
                        "listCost": {
                            "cost": 0,
                            "kubernetesPercent": 0
                        },
                        "netCost": {
                            "cost": 0,
                            "kubernetesPercent": 0
                        },
                        "amortizedNetCost": {
                            "cost": 0,
                            "kubernetesPercent": 0
                        },
                        "invoicedCost": {
                            "cost": 0,
                            "kubernetesPercent": 0
                        }
                    },
                    "AmazonAthena": {
                        "properties": {
                            "provider": "AWS",
                            "accountID": "297945954695",
                            "invoiceEntityID": "297945954695",
                            "service": "AmazonAthena"
                        },
                        "window": {
                            "start": "2023-04-30T00:00:00Z",
                            "end": "2023-05-01T00:00:00Z"
                        },
                        "listCost": {
                            "cost": 0.10061275,
                            "kubernetesPercent": 0
                        },
                        "netCost": {
                            "cost": 0.10061275,
                            "kubernetesPercent": 0
                        },
                        "amortizedNetCost": {
                            "cost": 0.10061275,
                            "kubernetesPercent": 0
                        },
                        "invoicedCost": {
                            "cost": 0.10061275,
                            "kubernetesPercent": 0
                        }
                    },
                    "AmazonCloudWatch": {
                        "properties": {
                            "provider": "AWS",
                            "accountID": "297945954695",
                            "invoiceEntityID": "297945954695",
                            "service": "AmazonCloudWatch"
                        },
                        "window": {
                            "start": "2023-04-30T00:00:00Z",
                            "end": "2023-05-01T00:00:00Z"
                        },
                        "listCost": {
                            "cost": 0.21150513669999998,
                            "kubernetesPercent": 0
                        },
                        "netCost": {
                            "cost": 0.21150513669999998,
                            "kubernetesPercent": 0
                        },
                        "amortizedNetCost": {
                            "cost": 0.21150513669999998,
                            "kubernetesPercent": 0
                        },
                        "invoicedCost": {
                            "cost": 0.21150513669999998,
                            "kubernetesPercent": 0
                        }
                    },
                    "AmazonEC2": {
                        "properties": {
                            "provider": "AWS",
                            "invoiceEntityID": "297945954695",
                            "service": "AmazonEC2"
                        },
                        "window": {
                            "start": "2023-04-30T00:00:00Z",
                            "end": "2023-05-01T00:00:00Z"
                        },
                        "listCost": {
                            "cost": 337.4926118030998,
                            "kubernetesPercent": 0.6543833295809984
                        },
                        "netCost": {
                            "cost": 337.4926118030998,
                            "kubernetesPercent": 0.6543833295809984
                        },
                        "amortizedNetCost": {
                            "cost": 337.4926118030998,
                            "kubernetesPercent": 0.6543833295809984
                        },
                        "invoicedCost": {
                            "cost": 337.4926118030998,
                            "kubernetesPercent": 0.6543833295809984
                        }
                    },
                    "AmazonECR": {
                        "properties": {
                            "provider": "AWS",
                            "accountID": "297945954695",
                            "invoiceEntityID": "297945954695",
                            "service": "AmazonECR"
                        },
                        "window": {
                            "start": "2023-04-30T00:00:00Z",
                            "end": "2023-05-01T00:00:00Z"
                        },
                        "listCost": {
                            "cost": 0.00018308879999999998,
                            "kubernetesPercent": 0
                        },
                        "netCost": {
                            "cost": 0.00018308879999999998,
                            "kubernetesPercent": 0
                        },
                        "amortizedNetCost": {
                            "cost": 0.00018308879999999998,
                            "kubernetesPercent": 0
                        },
                        "invoicedCost": {
                            "cost": 0.00018308879999999998,
                            "kubernetesPercent": 0
                        }
                    },
                    "AmazonECRPublic": {
                        "properties": {
                            "provider": "AWS",
                            "accountID": "297945954695",
                            "invoiceEntityID": "297945954695",
                            "service": "AmazonECRPublic"
                        },
                        "window": {
                            "start": "2023-04-30T00:00:00Z",
                            "end": "2023-05-01T00:00:00Z"
                        },
                        "listCost": {
                            "cost": 0,
                            "kubernetesPercent": 0
                        },
                        "netCost": {
                            "cost": 0,
                            "kubernetesPercent": 0
                        },
                        "amortizedNetCost": {
                            "cost": 0,
                            "kubernetesPercent": 0
                        },
                        "invoicedCost": {
                            "cost": 0,
                            "kubernetesPercent": 0
                        }
                    },
                    "AmazonEFS": {
                        "properties": {
                            "provider": "AWS",
                            "accountID": "297945954695",
                            "invoiceEntityID": "297945954695",
                            "service": "AmazonEFS"
                        },
                        "window": {
                            "start": "2023-04-30T00:00:00Z",
                            "end": "2023-05-01T00:00:00Z"
                        },
                        "listCost": {
                            "cost": 6.123e-07,
                            "kubernetesPercent": 0
                        },
                        "netCost": {
                            "cost": 6.123e-07,
                            "kubernetesPercent": 0
                        },
                        "amortizedNetCost": {
                            "cost": 6.123e-07,
                            "kubernetesPercent": 0
                        },
                        "invoicedCost": {
                            "cost": 6.123e-07,
                            "kubernetesPercent": 0
                        }
                    },
                    "AmazonEKS": {
                        "properties": {
                            "provider": "AWS",
                            "invoiceEntityID": "297945954695",
                            "service": "AmazonEKS"
                        },
                        "window": {
                            "start": "2023-04-30T00:00:00Z",
                            "end": "2023-05-01T00:00:00Z"
                        },
                        "listCost": {
                            "cost": 43.19999999999999,
                            "kubernetesPercent": 1
                        },
                        "netCost": {
                            "cost": 43.19999999999999,
                            "kubernetesPercent": 1
                        },
                        "amortizedNetCost": {
                            "cost": 43.19999999999999,
                            "kubernetesPercent": 1
                        },
                        "invoicedCost": {
                            "cost": 43.19999999999999,
                            "kubernetesPercent": 1
                        }
                    },
                    "AmazonFSx": {
                        "properties": {
                            "provider": "AWS",
                            "accountID": "297945954695",
                            "invoiceEntityID": "297945954695",
                            "service": "AmazonFSx"
                        },
                        "window": {
                            "start": "2023-04-30T00:00:00Z",
                            "end": "2023-05-01T00:00:00Z"
                        },
                        "listCost": {
                            "cost": 5.6010275086000005,
                            "kubernetesPercent": 0
                        },
                        "netCost": {
                            "cost": 5.6010275086000005,
                            "kubernetesPercent": 0
                        },
                        "amortizedNetCost": {
                            "cost": 5.6010275086000005,
                            "kubernetesPercent": 0
                        },
                        "invoicedCost": {
                            "cost": 5.6010275086000005,
                            "kubernetesPercent": 0
                        }
                    },
                    "AmazonPrometheus": {
                        "properties": {
                            "provider": "AWS",
                            "accountID": "297945954695",
                            "invoiceEntityID": "297945954695",
                            "service": "AmazonPrometheus"
                        },
                        "window": {
                            "start": "2023-04-30T00:00:00Z",
                            "end": "2023-05-01T00:00:00Z"
                        },
                        "listCost": {
                            "cost": 5.03357787,
                            "kubernetesPercent": 0
                        },
                        "netCost": {
                            "cost": 5.03357787,
                            "kubernetesPercent": 0
                        },
                        "amortizedNetCost": {
                            "cost": 5.03357787,
                            "kubernetesPercent": 0
                        },
                        "invoicedCost": {
                            "cost": 5.03357787,
                            "kubernetesPercent": 0
                        }
                    },
                    "AmazonQuickSight": {
                        "properties": {
                            "provider": "AWS",
                            "accountID": "297945954695",
                            "invoiceEntityID": "297945954695",
                            "service": "AmazonQuickSight"
                        },
                        "window": {
                            "start": "2023-04-30T00:00:00Z",
                            "end": "2023-05-01T00:00:00Z"
                        },
                        "listCost": {
                            "cost": 0.8000000064000001,
                            "kubernetesPercent": 0
                        },
                        "netCost": {
                            "cost": 0.8000000064000001,
                            "kubernetesPercent": 0
                        },
                        "amortizedNetCost": {
                            "cost": 0.8000000064000001,
                            "kubernetesPercent": 0
                        },
                        "invoicedCost": {
                            "cost": 0.8000000064000001,
                            "kubernetesPercent": 0
                        }
                    },
                    "AmazonRoute53": {
                        "properties": {
                            "provider": "AWS",
                            "accountID": "297945954695",
                            "invoiceEntityID": "297945954695",
                            "service": "AmazonRoute53"
                        },
                        "window": {
                            "start": "2023-04-30T00:00:00Z",
                            "end": "2023-05-01T00:00:00Z"
                        },
                        "listCost": {
                            "cost": 0.0005856,
                            "kubernetesPercent": 0
                        },
                        "netCost": {
                            "cost": 0.0005856,
                            "kubernetesPercent": 0
                        },
                        "amortizedNetCost": {
                            "cost": 0.0005856,
                            "kubernetesPercent": 0
                        },
                        "invoicedCost": {
                            "cost": 0.0005856,
                            "kubernetesPercent": 0
                        }
                    },
                    "AmazonS3": {
                        "properties": {
                            "provider": "AWS",
                            "invoiceEntityID": "297945954695",
                            "service": "AmazonS3"
                        },
                        "window": {
                            "start": "2023-04-30T00:00:00Z",
                            "end": "2023-05-01T00:00:00Z"
                        },
                        "listCost": {
                            "cost": 45.7935617916,
                            "kubernetesPercent": 0
                        },
                        "netCost": {
                            "cost": 45.7935617916,
                            "kubernetesPercent": 0
                        },
                        "amortizedNetCost": {
                            "cost": 45.7935617916,
                            "kubernetesPercent": 0
                        },
                        "invoicedCost": {
                            "cost": 45.7935617916,
                            "kubernetesPercent": 0
                        }
                    },
                    "AmazonSNS": {
                        "properties": {
                            "provider": "AWS",
                            "accountID": "297945954695",
                            "invoiceEntityID": "297945954695",
                            "service": "AmazonSNS"
                        },
                        "window": {
                            "start": "2023-04-30T00:00:00Z",
                            "end": "2023-05-01T00:00:00Z"
                        },
                        "listCost": {
                            "cost": 0,
                            "kubernetesPercent": 0
                        },
                        "netCost": {
                            "cost": 0,
                            "kubernetesPercent": 0
                        },
                        "amortizedNetCost": {
                            "cost": 0,
                            "kubernetesPercent": 0
                        },
                        "invoicedCost": {
                            "cost": 0,
                            "kubernetesPercent": 0
                        }
                    },
                    "AmazonVPC": {
                        "properties": {
                            "provider": "AWS",
                            "accountID": "297945954695",
                            "invoiceEntityID": "297945954695",
                            "service": "AmazonVPC"
                        },
                        "window": {
                            "start": "2023-04-30T00:00:00Z",
                            "end": "2023-05-01T00:00:00Z"
                        },
                        "listCost": {
                            "cost": 2.8800000000000017,
                            "kubernetesPercent": 0
                        },
                        "netCost": {
                            "cost": 2.8800000000000017,
                            "kubernetesPercent": 0
                        },
                        "amortizedNetCost": {
                            "cost": 2.8800000000000017,
                            "kubernetesPercent": 0
                        },
                        "invoicedCost": {
                            "cost": 2.8800000000000017,
                            "kubernetesPercent": 0
                        }
                    },
                    "awskms": {
                        "properties": {
                            "provider": "AWS",
                            "accountID": "297945954695",
                            "invoiceEntityID": "297945954695",
                            "service": "awskms"
                        },
                        "window": {
                            "start": "2023-04-30T00:00:00Z",
                            "end": "2023-05-01T00:00:00Z"
                        },
                        "listCost": {
                            "cost": 0.23333333520000016,
                            "kubernetesPercent": 0
                        },
                        "netCost": {
                            "cost": 0.23333333520000016,
                            "kubernetesPercent": 0
                        },
                        "amortizedNetCost": {
                            "cost": 0.23333333520000016,
                            "kubernetesPercent": 0
                        },
                        "invoicedCost": {
                            "cost": 0.23333333520000016,
                            "kubernetesPercent": 0
                        }
                    }
                },
                "window": {
                    "start": "2023-04-30T00:00:00Z",
                    "end": "2023-05-01T00:00:00Z"
                },
                "aggregationProperties": [
                    "service"
                ]
            },
            {
                "cloudCosts": {
                    "5hnnev4d0v7mapf09j0v8of0o2": {
                        "properties": {
                            "provider": "AWS",
                            "accountID": "297945954695",
                            "invoiceEntityID": "297945954695",
                            "service": "5hnnev4d0v7mapf09j0v8of0o2"
                        },
                        "window": {
                            "start": "2023-05-01T00:00:00Z",
                            "end": "2023-05-02T00:00:00Z"
                        },
                        "listCost": {
                            "cost": 7.865999999999996,
                            "kubernetesPercent": 0
                        },
                        "netCost": {
                            "cost": 7.865999999999996,
                            "kubernetesPercent": 0
                        },
                        "amortizedNetCost": {
                            "cost": 7.865999999999996,
                            "kubernetesPercent": 0
                        },
                        "invoicedCost": {
                            "cost": 7.865999999999996,
                            "kubernetesPercent": 0
                        }
                    },
                    "AWSCloudTrail": {
                        "properties": {
                            "provider": "AWS",
                            "accountID": "297945954695",
                            "invoiceEntityID": "297945954695",
                            "service": "AWSCloudTrail"
                        },
                        "window": {
                            "start": "2023-05-01T00:00:00Z",
                            "end": "2023-05-02T00:00:00Z"
                        },
                        "listCost": {
                            "cost": 6.373088000000007,
                            "kubernetesPercent": 0
                        },
                        "netCost": {
                            "cost": 6.373088000000007,
                            "kubernetesPercent": 0
                        },
                        "amortizedNetCost": {
                            "cost": 6.373088000000007,
                            "kubernetesPercent": 0
                        },
                        "invoicedCost": {
                            "cost": 6.373088000000007,
                            "kubernetesPercent": 0
                        }
                    },
                    "AWSCostExplorer": {
                        "properties": {
                            "provider": "AWS",
                            "accountID": "297945954695",
                            "invoiceEntityID": "297945954695",
                            "service": "AWSCostExplorer"
                        },
                        "window": {
                            "start": "2023-05-01T00:00:00Z",
                            "end": "2023-05-02T00:00:00Z"
                        },
                        "listCost": {
                            "cost": 0.24415709680000003,
                            "kubernetesPercent": 0
                        },
                        "netCost": {
                            "cost": 0.24415709680000003,
                            "kubernetesPercent": 0
                        },
                        "amortizedNetCost": {
                            "cost": 0.24415709680000003,
                            "kubernetesPercent": 0
                        },
                        "invoicedCost": {
                            "cost": 0.24415709680000003,
                            "kubernetesPercent": 0
                        }
                    },
                    "AWSELB": {
                        "properties": {
                            "provider": "AWS",
                            "invoiceEntityID": "297945954695",
                            "service": "AWSELB"
                        },
                        "window": {
                            "start": "2023-05-01T00:00:00Z",
                            "end": "2023-05-02T00:00:00Z"
                        },
                        "listCost": {
                            "cost": 41.16003439479998,
                            "kubernetesPercent": 0.8082905243733983
                        },
                        "netCost": {
                            "cost": 41.16003439479998,
                            "kubernetesPercent": 0.8082905243733983
                        },
                        "amortizedNetCost": {
                            "cost": 41.16003439479998,
                            "kubernetesPercent": 0.8082905243733983
                        },
                        "invoicedCost": {
                            "cost": 41.16003439479998,
                            "kubernetesPercent": 0.8082905243733983
                        }
                    },
                    "AWSGlue": {
                        "properties": {
                            "provider": "AWS",
                            "accountID": "297945954695",
                            "invoiceEntityID": "297945954695",
                            "service": "AWSGlue"
                        },
                        "window": {
                            "start": "2023-05-01T00:00:00Z",
                            "end": "2023-05-02T00:00:00Z"
                        },
                        "listCost": {
                            "cost": 0.5083949200000001,
                            "kubernetesPercent": 0
                        },
                        "netCost": {
                            "cost": 0.5083949200000001,
                            "kubernetesPercent": 0
                        },
                        "amortizedNetCost": {
                            "cost": 0.5083949200000001,
                            "kubernetesPercent": 0
                        },
                        "invoicedCost": {
                            "cost": 0.5083949200000001,
                            "kubernetesPercent": 0
                        }
                    },
                    "AWSLambda": {
                        "properties": {
                            "provider": "AWS",
                            "accountID": "297945954695",
                            "invoiceEntityID": "297945954695",
                            "service": "AWSLambda"
                        },
                        "window": {
                            "start": "2023-05-01T00:00:00Z",
                            "end": "2023-05-02T00:00:00Z"
                        },
                        "listCost": {
                            "cost": 0,
                            "kubernetesPercent": 0
                        },
                        "netCost": {
                            "cost": 0,
                            "kubernetesPercent": 0
                        },
                        "amortizedNetCost": {
                            "cost": 0,
                            "kubernetesPercent": 0
                        },
                        "invoicedCost": {
                            "cost": 0,
                            "kubernetesPercent": 0
                        }
                    },
                    "AWSQueueService": {
                        "properties": {
                            "provider": "AWS",
                            "accountID": "297945954695",
                            "invoiceEntityID": "297945954695",
                            "service": "AWSQueueService"
                        },
                        "window": {
                            "start": "2023-05-01T00:00:00Z",
                            "end": "2023-05-02T00:00:00Z"
                        },
                        "listCost": {
                            "cost": 0,
                            "kubernetesPercent": 0
                        },
                        "netCost": {
                            "cost": 0,
                            "kubernetesPercent": 0
                        },
                        "amortizedNetCost": {
                            "cost": 0,
                            "kubernetesPercent": 0
                        },
                        "invoicedCost": {
                            "cost": 0,
                            "kubernetesPercent": 0
                        }
                    },
                    "AmazonAthena": {
                        "properties": {
                            "provider": "AWS",
                            "accountID": "297945954695",
                            "invoiceEntityID": "297945954695",
                            "service": "AmazonAthena"
                        },
                        "window": {
                            "start": "2023-05-01T00:00:00Z",
                            "end": "2023-05-02T00:00:00Z"
                        },
                        "listCost": {
                            "cost": 10.695624500000003,
                            "kubernetesPercent": 0
                        },
                        "netCost": {
                            "cost": 10.695624500000003,
                            "kubernetesPercent": 0
                        },
                        "amortizedNetCost": {
                            "cost": 10.695624500000003,
                            "kubernetesPercent": 0
                        },
                        "invoicedCost": {
                            "cost": 10.695624500000003,
                            "kubernetesPercent": 0
                        }
                    },
                    "AmazonCloudWatch": {
                        "properties": {
                            "provider": "AWS",
                            "accountID": "297945954695",
                            "invoiceEntityID": "297945954695",
                            "service": "AmazonCloudWatch"
                        },
                        "window": {
                            "start": "2023-05-01T00:00:00Z",
                            "end": "2023-05-02T00:00:00Z"
                        },
                        "listCost": {
                            "cost": 0.0148635813,
                            "kubernetesPercent": 0
                        },
                        "netCost": {
                            "cost": 0.0148635813,
                            "kubernetesPercent": 0
                        },
                        "amortizedNetCost": {
                            "cost": 0.0148635813,
                            "kubernetesPercent": 0
                        },
                        "invoicedCost": {
                            "cost": 0.0148635813,
                            "kubernetesPercent": 0
                        }
                    },
                    "AmazonEC2": {
                        "properties": {
                            "provider": "AWS",
                            "invoiceEntityID": "297945954695",
                            "service": "AmazonEC2"
                        },
                        "window": {
                            "start": "2023-05-01T00:00:00Z",
                            "end": "2023-05-02T00:00:00Z"
                        },
                        "listCost": {
                            "cost": 309.4241635897003,
                            "kubernetesPercent": 0.6593596481215193
                        },
                        "netCost": {
                            "cost": 309.4241635897003,
                            "kubernetesPercent": 0.6593596481215193
                        },
                        "amortizedNetCost": {
                            "cost": 309.4241635897003,
                            "kubernetesPercent": 0.6593596481215193
                        },
                        "invoicedCost": {
                            "cost": 309.4241635897003,
                            "kubernetesPercent": 0.6593596481215193
                        }
                    },
                    "AmazonECR": {
                        "properties": {
                            "provider": "AWS",
                            "accountID": "297945954695",
                            "invoiceEntityID": "297945954695",
                            "service": "AmazonECR"
                        },
                        "window": {
                            "start": "2023-05-01T00:00:00Z",
                            "end": "2023-05-02T00:00:00Z"
                        },
                        "listCost": {
                            "cost": 0.00014835589999999998,
                            "kubernetesPercent": 0
                        },
                        "netCost": {
                            "cost": 0.00014835589999999998,
                            "kubernetesPercent": 0
                        },
                        "amortizedNetCost": {
                            "cost": 0.00014835589999999998,
                            "kubernetesPercent": 0
                        },
                        "invoicedCost": {
                            "cost": 0.00014835589999999998,
                            "kubernetesPercent": 0
                        }
                    },
                    "AmazonECRPublic": {
                        "properties": {
                            "provider": "AWS",
                            "accountID": "297945954695",
                            "invoiceEntityID": "297945954695",
                            "service": "AmazonECRPublic"
                        },
                        "window": {
                            "start": "2023-05-01T00:00:00Z",
                            "end": "2023-05-02T00:00:00Z"
                        },
                        "listCost": {
                            "cost": 0,
                            "kubernetesPercent": 0
                        },
                        "netCost": {
                            "cost": 0,
                            "kubernetesPercent": 0
                        },
                        "amortizedNetCost": {
                            "cost": 0,
                            "kubernetesPercent": 0
                        },
                        "invoicedCost": {
                            "cost": 0,
                            "kubernetesPercent": 0
                        }
                    },
                    "AmazonEFS": {
                        "properties": {
                            "provider": "AWS",
                            "accountID": "297945954695",
                            "invoiceEntityID": "297945954695",
                            "service": "AmazonEFS"
                        },
                        "window": {
                            "start": "2023-05-01T00:00:00Z",
                            "end": "2023-05-02T00:00:00Z"
                        },
                        "listCost": {
                            "cost": 5.681000000000001e-07,
                            "kubernetesPercent": 0
                        },
                        "netCost": {
                            "cost": 5.681000000000001e-07,
                            "kubernetesPercent": 0
                        },
                        "amortizedNetCost": {
                            "cost": 5.681000000000001e-07,
                            "kubernetesPercent": 0
                        },
                        "invoicedCost": {
                            "cost": 5.681000000000001e-07,
                            "kubernetesPercent": 0
                        }
                    },
                    "AmazonEKS": {
                        "properties": {
                            "provider": "AWS",
                            "invoiceEntityID": "297945954695",
                            "service": "AmazonEKS"
                        },
                        "window": {
                            "start": "2023-05-01T00:00:00Z",
                            "end": "2023-05-02T00:00:00Z"
                        },
                        "listCost": {
                            "cost": 39.6,
                            "kubernetesPercent": 1
                        },
                        "netCost": {
                            "cost": 39.6,
                            "kubernetesPercent": 1
                        },
                        "amortizedNetCost": {
                            "cost": 39.6,
                            "kubernetesPercent": 1
                        },
                        "invoicedCost": {
                            "cost": 39.6,
                            "kubernetesPercent": 1
                        }
                    },
                    "AmazonFSx": {
                        "properties": {
                            "provider": "AWS",
                            "accountID": "297945954695",
                            "invoiceEntityID": "297945954695",
                            "service": "AmazonFSx"
                        },
                        "window": {
                            "start": "2023-05-01T00:00:00Z",
                            "end": "2023-05-02T00:00:00Z"
                        },
                        "listCost": {
                            "cost": 4.968756381500007,
                            "kubernetesPercent": 0
                        },
                        "netCost": {
                            "cost": 4.968756381500007,
                            "kubernetesPercent": 0
                        },
                        "amortizedNetCost": {
                            "cost": 4.968756381500007,
                            "kubernetesPercent": 0
                        },
                        "invoicedCost": {
                            "cost": 4.968756381500007,
                            "kubernetesPercent": 0
                        }
                    },
                    "AmazonPrometheus": {
                        "properties": {
                            "provider": "AWS",
                            "accountID": "297945954695",
                            "invoiceEntityID": "297945954695",
                            "service": "AmazonPrometheus"
                        },
                        "window": {
                            "start": "2023-05-01T00:00:00Z",
                            "end": "2023-05-02T00:00:00Z"
                        },
                        "listCost": {
                            "cost": 1.04940423,
                            "kubernetesPercent": 0
                        },
                        "netCost": {
                            "cost": 1.04940423,
                            "kubernetesPercent": 0
                        },
                        "amortizedNetCost": {
                            "cost": 1.04940423,
                            "kubernetesPercent": 0
                        },
                        "invoicedCost": {
                            "cost": 1.04940423,
                            "kubernetesPercent": 0
                        }
                    },
                    "AmazonQuickSight": {
                        "properties": {
                            "provider": "AWS",
                            "accountID": "297945954695",
                            "invoiceEntityID": "297945954695",
                            "service": "AmazonQuickSight"
                        },
                        "window": {
                            "start": "2023-05-01T00:00:00Z",
                            "end": "2023-05-02T00:00:00Z"
                        },
                        "listCost": {
                            "cost": 0.7419354719999997,
                            "kubernetesPercent": 0
                        },
                        "netCost": {
                            "cost": 0.7419354719999997,
                            "kubernetesPercent": 0
                        },
                        "amortizedNetCost": {
                            "cost": 0.7419354719999997,
                            "kubernetesPercent": 0
                        },
                        "invoicedCost": {
                            "cost": 0.7419354719999997,
                            "kubernetesPercent": 0
                        }
                    },
                    "AmazonRoute53": {
                        "properties": {
                            "provider": "AWS",
                            "accountID": "297945954695",
                            "invoiceEntityID": "297945954695",
                            "service": "AmazonRoute53"
                        },
                        "window": {
                            "start": "2023-05-01T00:00:00Z",
                            "end": "2023-05-02T00:00:00Z"
                        },
                        "listCost": {
                            "cost": 1.5010184,
                            "kubernetesPercent": 0
                        },
                        "netCost": {
                            "cost": 1.5010184,
                            "kubernetesPercent": 0
                        },
                        "amortizedNetCost": {
                            "cost": 1.5010184,
                            "kubernetesPercent": 0
                        },
                        "invoicedCost": {
                            "cost": 1.5010184,
                            "kubernetesPercent": 0
                        }
                    },
                    "AmazonS3": {
                        "properties": {
                            "provider": "AWS",
                            "accountID": "297945954695",
                            "invoiceEntityID": "297945954695",
                            "service": "AmazonS3"
                        },
                        "window": {
                            "start": "2023-05-01T00:00:00Z",
                            "end": "2023-05-02T00:00:00Z"
                        },
                        "listCost": {
                            "cost": 35.486366779799866,
                            "kubernetesPercent": 0
                        },
                        "netCost": {
                            "cost": 35.486366779799866,
                            "kubernetesPercent": 0
                        },
                        "amortizedNetCost": {
                            "cost": 35.486366779799866,
                            "kubernetesPercent": 0
                        },
                        "invoicedCost": {
                            "cost": 35.486366779799866,
                            "kubernetesPercent": 0
                        }
                    },
                    "AmazonSNS": {
                        "properties": {
                            "provider": "AWS",
                            "accountID": "297945954695",
                            "invoiceEntityID": "297945954695",
                            "service": "AmazonSNS"
                        },
                        "window": {
                            "start": "2023-05-01T00:00:00Z",
                            "end": "2023-05-02T00:00:00Z"
                        },
                        "listCost": {
                            "cost": 0,
                            "kubernetesPercent": 0
                        },
                        "netCost": {
                            "cost": 0,
                            "kubernetesPercent": 0
                        },
                        "amortizedNetCost": {
                            "cost": 0,
                            "kubernetesPercent": 0
                        },
                        "invoicedCost": {
                            "cost": 0,
                            "kubernetesPercent": 0
                        }
                    },
                    "AmazonVPC": {
                        "properties": {
                            "provider": "AWS",
                            "accountID": "297945954695",
                            "invoiceEntityID": "297945954695",
                            "service": "AmazonVPC"
                        },
                        "window": {
                            "start": "2023-05-01T00:00:00Z",
                            "end": "2023-05-02T00:00:00Z"
                        },
                        "listCost": {
                            "cost": 2.849999999999996,
                            "kubernetesPercent": 0
                        },
                        "netCost": {
                            "cost": 2.849999999999996,
                            "kubernetesPercent": 0
                        },
                        "amortizedNetCost": {
                            "cost": 2.849999999999996,
                            "kubernetesPercent": 0
                        },
                        "invoicedCost": {
                            "cost": 2.849999999999996,
                            "kubernetesPercent": 0
                        }
                    },
                    "AmazonWorkSpaces": {
                        "properties": {
                            "provider": "AWS",
                            "accountID": "297945954695",
                            "invoiceEntityID": "297945954695",
                            "service": "AmazonWorkSpaces"
                        },
                        "window": {
                            "start": "2023-05-01T00:00:00Z",
                            "end": "2023-05-02T00:00:00Z"
                        },
                        "listCost": {
                            "cost": 38,
                            "kubernetesPercent": 0
                        },
                        "netCost": {
                            "cost": 38,
                            "kubernetesPercent": 0
                        },
                        "amortizedNetCost": {
                            "cost": 38,
                            "kubernetesPercent": 0
                        },
                        "invoicedCost": {
                            "cost": 38,
                            "kubernetesPercent": 0
                        }
                    },
                    "awskms": {
                        "properties": {
                            "provider": "AWS",
                            "accountID": "297945954695",
                            "invoiceEntityID": "297945954695",
                            "service": "awskms"
                        },
                        "window": {
                            "start": "2023-05-01T00:00:00Z",
                            "end": "2023-05-02T00:00:00Z"
                        },
                        "listCost": {
                            "cost": 0.2163978459999994,
                            "kubernetesPercent": 0
                        },
                        "netCost": {
                            "cost": 0.2163978459999994,
                            "kubernetesPercent": 0
                        },
                        "amortizedNetCost": {
                            "cost": 0.2163978459999994,
                            "kubernetesPercent": 0
                        },
                        "invoicedCost": {
                            "cost": 0.2163978459999994,
                            "kubernetesPercent": 0
                        }
                    }
                },
                "window": {
                    "start": "2023-05-01T00:00:00Z",
                    "end": "2023-05-02T00:00:00Z"
                },
                "aggregationProperties": [
                    "service"
                ]
            },
            {
                "cloudCosts": {},
                "window": {
                    "start": "2023-05-02T00:00:00Z",
                    "end": "2023-05-03T00:00:00Z"
                },
                "aggregationProperties": [
                    "service"
                ]
            }
        ],
        "window": {
            "start": "2023-04-30T00:00:00Z",
            "end": "2023-05-03T00:00:00Z"
        }
    }
}
```

```